processor/memorylimiterprocessor/config.go

```diff
@@ -51,9 +51,34 @@ type Config struct {
 	MemorySpikePercentage uint32 `mapstructure:"spike_limit_percentage"`
 }
 
-var _ component.Config = (*Config)(nil)
+var (
+	_ component.Config = (*Config)(nil)
+)
 
 // Validate checks if the processor configuration is valid
 func (cfg *Config) Validate() error {
```

Comment:
Were all this conditions being checked at runtime, or is this a breaking change?

Reply:
Yeah, they were being checked within the `Start` function